# Vincent FOUCAULT
# avril 2016
####################

string msg
